### PR TITLE
User profile orders link#40

### LINKED ISSUE #40: User Story 32, User Profile displays Orders link
```
As a registered user
When I visit my Profile page
And I have orders placed in the system
Then I see a link to my Profile Orders page

This link is also in the nav bar, but this new link is for admins.
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #41

child of #79

depends on #57

### PR DESCRIPTION
**Pull request by timnallen**
As a registered user
When I visit my Profile page
And I have orders placed in the system
Then I see a link to my Profile Orders page

This link is also in the nav bar, but this new link is for admins.

closes #40

### CODE REVIEW

**Review by allieIsOkay — APPROVED** [review]

File: spec/features/admins/admin_sees_merchant_dashboard_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    it 'takes me to the route admin/merchants/:id' do
+      visit merchants_path
+
+      click_on "#{@merchant.name}"
+
+      expect(current_path).to eq(admin_merchant_path(@merchant))
+    end
+
+    # it 'shows the same info the merchant sees' do
```
Comment (csvlewis) [inline spec/features/admins/admin_sees_merchant_dashboard_spec.rb:20]: Why is this test commented out? Is it for a feature that hasn't been written yet?

**Review by csvlewis — APPROVED** [review]
Besides the question I had about the commented out test, this looks good!

File: spec/features/admins/admin_sees_merchant_dashboard_spec.rb (hunk 1 above)
Comment (timnallen) [reply spec/features/admins/admin_sees_merchant_dashboard_spec.rb:20]: Yes this whole file is the first half of a user story I started on a different branch, but I needed the merchant dashboard to complete. I'll finish it as soon as I can merge the merchant dashboard.

**Review by teresa-m-knowles — APPROVED** [review]
Looks good, Tim
